G-Chilie/Banka-UI
# #165482848 Build out REST Endpoint for Signup (CRUD)

#### What does this PR do?

Implements user signup functionality

#### Description of Task to be completed?

have this endpoint working
- POST /api/v1/auth/signup

#### How should this be manually tested?

- Clone https://github.com/G-Chilie/Banka.git
-  On your terminal cd into the banka folder
-  Run npm install
-  Run npm run before:start
-  Run npm dev
-  On postman make a POST request to _http://localhost:3000/api/v1/auth/signup
NB see screenshot below on the request data to pass

#### What are the relevant pivotal tracker stories?

[#165482848 
](https://www.pivotaltracker.com/story/show/165482848)

#### Screenshots (if appropriate)

![signup-user](https://user-images.githubusercontent.com/39604096/56536481-17a82880-6556-11e9-89f0-8019e458cdef.png)
